src/containers/EditProfile.vue

```diff
@@ -80,11 +105,71 @@ export default defineComponent({
         ? await blobToDataURL(resizedImage!)
         : undefined;
 
+      const userPerspective = this.userStore.getFluxPerspectiveId;
+
+      const langs = await ad4mClient.languages.all();
+      const ipfsLang = langs.find(
+        (e) => e.address === NOTE_IPFS_EXPRESSION_OFFICIAL
+      );
+
+      console.log("langs", ipfsLang);
+
+      const image = await ad4mClient.expression.create(
+        this.profileBg,
+        NOTE_IPFS_EXPRESSION_OFFICIAL
+      );
+
+      console.log("langs", this.profileBg, image);
+
+      const linked = await ad4mClient.perspective.addLink(
+        userPerspective!,
+        new Link({
+          source: "flux://profile",
+          target: `text://${this.bio}`,
+          predicate: "sioc://has_bio",
+        })
+      );
+
+      const profileBgLinked = await ad4mClient.perspective.addLink(
+        userPerspective!,
+        new Link({
+          source: "flux://profile",
+          target: `image://${image}`,
+          predicate: "sioc://has_image",
+        })
+      );
```

Comment:
We shouldn't add a new link each time the profile is updated since links are unique even if the subject, predicate, target are the same (due to timestamp + signature that gets added) we should only add the link which is required during this update and also be sure to remove any existing links of the same type.